### DISCUSSION TITLE
Duda respecto a ENTREGAR-PEDIDOS

### DISCUSSION THREAD

**Discussion by jalonsoluc** [IIC2133-PUC/2024-1#21] [Tarea 0] [answered]
Hola!

El comando ENTREGAR-PEDIDOS debe hacer que todas las naves entreguen **todos sus pedidos**? O **sólo los pedidos del planeta correspondiente pedido más prioritario**? 

En otras palabras, al ejecutar el comando, ¿las naves tienen que viajar a un sólo planeta? ¿O a todos los que corresponda según sus pedidos?

Gracias!
Juan José

**Comment by clumsypunch** [comment]
Por comando es un viaje y entrega por nave en su planeta prioritario.

**Comment by PabloNavarroY** [comment, marked as answer]
Todas las naves viajan al planeta del pedido más urgente que tienen y aprovechan de entregar además todos los pedidos que pueden en ese planeta. No entregan otro paquete además de eso.
